pyspecProcScripts/correlation_alignment.py

```diff
@@ -117,47 +117,55 @@ def correl_align(
     energy_diff = 1.0
     i = 0
     energy_vals = []
-    this_E = (abs(s_orig.C.sum(indirect_dim)) ** 2).data.sum().item() / N**2
+    this_E = (abs(s_jk.C.sum(indirect_dim)) ** 2).data.sum().item() / N**2
     energy_vals.append(this_E / sig_energy)
     last_E = None
-    for_nu_center = s_orig.C
+    # {{{ find center frequency to see where to center the mask
+    for_nu_center = s_jk.C
     for_nu_center.ft(list(signal_pathway))
     for x in range(len(signal_keys)):
         for_nu_center = for_nu_center[signal_keys[x], signal_values[x]]
     nu_center = for_nu_center.mean(indirect_dim).C.argmax(direct)
+    # }}}
     logging.debug(psd.strm("Center frequency", nu_center))
     for my_iter in range(100):
         i += 1
         logging.debug(psd.strm("*** *** ***"))
         logging.debug(psd.strm("CORRELATION ALIGNMENT ITERATION NO. ", i))
         logging.debug(psd.strm("*** *** ***"))
-        s_orig.ift(direct)
-        s_copy = s_orig.C
+        s_jk.ift(direct)
+        # s_mn will become the signal with the additional Delta ph# dimension.
+        s_mn = s_jk.C
```

Comment:
made a comment noting that we are now making s_mn. I make a copy though because later we will want to use s_jk still as is

Reply:
do not make a copy here.  One of them is replicated across the Δφ dimension.  The multiplication that is used to generate the replicas creates a copy for you.